os.bpmsuite.elasticsearch/added/openshift-launch.sh

```diff
@@ -1,13 +1,13 @@
 #!/bin/sh
-# Elasticsearch launch script
+# Openshift BPMS Elasticsearch launch script
```

Comment:
Just to keep consistency with the names across the scripts, use BPM Suite like in configure.sh script.